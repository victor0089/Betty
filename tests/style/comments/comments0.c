/* This comment is OK */

// This comment should not be used

/*
 * This is the preferred style for multi-line
 * comments in the Linux kernel source code.
 * Please use it consistently.
 *
 * Description:  A column of asterisks on the left side,
 * with beginning and ending almost-blank lines.
 */

/* Bad
 * multi-line
 * comment */

/** Also
very bad
comment*/
