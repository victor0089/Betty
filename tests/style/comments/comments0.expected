tests/style/comments/comments0.c:3: ERROR: do not use C99 // comments
tests/style/comments/comments0.c:14: WARNING: Block comments use a leading /* on a separate line
tests/style/comments/comments0.c:16: WARNING: Block comments use a trailing */ on a separate line
tests/style/comments/comments0.c:18: WARNING: Block comments use a leading /* on a separate line
tests/style/comments/comments0.c:19: WARNING: Block comments use * on subsequent lines
tests/style/comments/comments0.c:20: WARNING: Block comments use a trailing */ on a separate line
total: 1 errors, 5 warnings, 20 lines checked
